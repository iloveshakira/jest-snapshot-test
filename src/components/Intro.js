import React, {Component} from 'react';
import '../App.css';

class Intro extends Component {
	render(){
		return (
			<p className="App-intro">
				to get started, edit <code>src/app.js</code>
			</p>
			);
	}
}

export default Intro;