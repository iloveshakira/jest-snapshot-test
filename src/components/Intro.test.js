import React, {Component} from 'react';
const ReactTestRenderer = require('react-test-renderer');
import Intro from './Intro.js';


describe('Our first snapshot test', ()=>{
	it('Should compare the component with a snapshot', ()=>{
	const component = ReactTestRenderer.create(
		<Intro />
		); 
	const json= component.toJSON();
	expect(json).toMatchSnapshot();
		})
});