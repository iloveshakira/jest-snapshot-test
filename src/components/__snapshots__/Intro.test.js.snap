exports[`Our first snapshot test Should compare the component with a snapshot 1`] = `
<p
  className="App-intro">
  to get started, edit 
  <code>
    src/app.js
  </code>
</p>
`;
